pom.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+<modelVersion>4.0.0</modelVersion>
+<groupId>com.redhat.rhjmc.containerjfr</groupId>
+<artifactId>container-jfr</artifactId>
+<version>0.16.0</version>
+<packaging>jar</packaging>
+<name>container-jfr</name>
+<url>http://maven.apache.org</url>
+
+<properties>
+  <java.version>11</java.version>
+  <maven.compiler.target>${java.version}</maven.compiler.target>
+  <maven.compiler.source>${java.version}</maven.compiler.source>
+  <imageBuilder>/usr/bin/podman</imageBuilder>
+  <containerjfr.minimal>false</containerjfr.minimal>
+  <containerjfr.imageStream>quay.io/rh-jmc-team/container-jfr</containerjfr.imageStream>
+  <containerjfr.webPort>8181</containerjfr.webPort>
+  <containerjfr.listenPort>9090</containerjfr.listenPort>
```

Comment:
Adding `<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>` to properties prevents a few warnings about guessing the encoding.